### PR TITLE
Update with-exchange-online.md

### PR DESCRIPTION
**Pull request by pupara**
Added Notes
Replaced SkypeOnlineConnector with MicrosoftTeams

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/with-exchange-online.md [hunk 1]
```diff
@@ -114,8 +114,12 @@ If you deployed Active Directory Federation Services (AD FS), you may have to co
 
 1. Create a remote Windows PowerShell session from a PC as follows:
 
+> [!NOTE]
+> Skype for Business Online Connector is currently part of latest Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline Teams/rooms/with-exchange-online.md:118]:
```suggestion
> The Skype for Business Online Connector is currently part of the latest Teams PowerShell module.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**